wikichange/src/content/content.js

```diff
@@ -264,20 +261,7 @@ const renderItemsBelowGraph = async (creationDate) => {
         );
 
         highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
-        revisionButton.disabled = false;
-    });
-
-    revisionButton.addEventListener("click", async () => {
-        try {
-            window.open(getRevisionPageLink(title, curRevisionId, oldRevisionId), "_blank");
-        } catch (err) {
-            console.error(
-                `Error getting revision link between revision ids for inputs title:${title} curRevisionId:${curRevisionId} oldRevisionId:${oldRevisionId}\nError: ${err}`
-            );
-        }
     });
-
-    return [curRevisionId, oldRevisionId]
```

Comment:
need this return, and it will start highlighting after you remove the `highlightRevisionBetweenRevisionIds` above